Applications/WebPortal/WebApp/tests/entitlements_page_tester/source/index.tsx

```diff
@@ -3,14 +3,11 @@ import * as Nexus from 'nexus';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import * as WebPortal from 'web_portal';
-import { EntitlementsPage } from 'web_portal';
-import { CheckMarkButton } from 'web_portal';
-import { DropDownButton } from 'web_portal';
 
 const buttontest = (
   <div>
-    <CheckMarkButton size='50px'isReadOnly={true}/>
-    <DropDownButton size='50px'/>
+    <WebPortal.CheckMarkButton size='50px'isReadOnly={true}/>
```

Comment:
Fix space before ```isReadOnly```. Also when possible for ```boolean``` props you don't need to assign it, simply including it will set it to true, ie ```<Element size='50px' isReadOnly/>```